test/RadworksGovernor.t.sol

```diff
@@ -652,9 +666,475 @@ abstract contract Propose is ProposalTest {
+    // Ensure proposal can be queued after success
+    uint8 _state = governorAlpha.state(upgradeProposalId);
+    assertEq(_state, QUEUED);
+
+    (
+      address[] memory _targets,
+      uint256[] memory _values,
+      string[] memory _signatures,
+      bytes[] memory _calldatas
+    ) = governorAlpha.getActions(upgradeProposalId);
+
+    uint256 _eta = block.timestamp + timelock.delay();
+
+    for (uint256 _index = 0; _index < _targets.length; _index++) {
+      // Calculate hash of transaction in Timelock
+      bytes32 _txHash = keccak256(
+        abi.encode(_targets[_index], _values[_index], _signatures[_index], _calldatas[_index], _eta)
+      );
+
+      // Ensure transaction is queued in Timelock
+      bool _isQueued = timelock.queuedTransactions(_txHash);
+      assertEq(_isQueued, true);
+    }
+  }
+
+  function test_UpgradeProposalCanBeExecutedAfterDelay() public {
+    _passAndQueueUpgradeProposal();
+    _jumpPastProposalEta();
+
+    // Execute the proposal
+    governorAlpha.execute(upgradeProposalId);
+
+    // Ensure the proposal is now executed
+    uint8 _state = governorAlpha.state(upgradeProposalId);
+    assertEq(_state, EXECUTED);
+
+    // Ensure the governorBravo is now the admin of the timelock
+    assertEq(timelock.admin(), address(governorBravo));
+  }
+
+  //
+  // Post proposal tests
+  //
+  function testFuzz_OldGovernorSendsETHAfterProposalIsDefeated(uint128 _amount, address _receiver)
+    public
+  {
+    _assumeReceiver(_receiver);
+
+    // Counter-intuitively, the Governor (not the Timelock) must hold the ETH.
+    // See the deployed GovernorAlpha, line 281:
+    //   https://etherscan.io/address/0x690e775361AD66D1c4A25d89da9fCd639F5198eD#code
+    // The governor transfers ETH to the Timelock in the process of executing
+    // the proposal. The Timelock then just passes that ETH along.
+    vm.deal(address(governorAlpha), _amount);
+
+    uint256 _receiverETHBalance = _receiver.balance;
+    uint256 _governorETHBalance = address(governorAlpha).balance;
+
+    // Defeat the proposal to upgrade the Governor
+    _defeatUpgradeProposal();
+
+    // Create a new proposal to send the ETH.
+    address[] memory _targets = new address[](1);
+    uint256[] memory _values = new uint256[](1);
+    _targets[0] = _receiver;
+    _values[0] = _amount;
+
+    _queueAndVoteAndExecuteProposalWithAlphaGovernor(
+      _targets,
+      _values,
+      new string[](1), // No signature needed for an ETH send.
+      new bytes[](1), // No calldata needed for an ETH send.
+      true // GovernorAlpha is still the Timelock admin.
+    );
+
+    // Ensure the ETH has been transferred to the receiver
+    assertEq(
+      address(governorAlpha).balance,
+      _governorETHBalance - _amount,
+      "Governor alpha ETH balance is incorrect"
+    );
+    assertEq(_receiver.balance, _receiverETHBalance + _amount, "Receiver ETH balance is incorrect");
+  }
+
+  function testFuzz_OldGovernorCannotSendETHAfterProposalSucceeds(
```

Comment:
these lines are all marking as new in the diff. Did anything in these tests change besides the names?

Reply:
Yes.. these came from the file RadworksGovernorAlpha.t.sol file.

Reply:
Thanks @jferas , I mean did the tests change besides the relocation and the test name changes? if not, you have my approval for merge

Reply:
No only the names and locations of the tests have changed.